services/remotecluster/invitation.go

```diff
@@ -51,8 +51,12 @@ func (rcs *Service) AcceptInvitation(invite *model.RemoteClusterInvite, name str
 
 	if !response.IsSuccess() {
 		rcs.server.GetStore().RemoteCluster().Delete(rcSaved.RemoteId)
-		return nil, errors.New(response.Error())
+		return nil, errors.New(response.Err)
 	}
+
+	// issue the first ping right away. The goroutine will exit when ping completes or PingTimeout exceeded.
+	go rcs.pingRemote(rcSaved)
```

Comment:
I really only did this for demos/testing where you create an invite and then immediately paste it into another browser.  I guess some admins inviting clusters within their own org might do this as well.  You want to immediately share a channel with them so they should get marked online right away.  In the other direction it doesn't matter so much - the average 30 sec wait for first ping won't matter.

Also, when accepting an invitation a confirm message is sent to the issuer's `/remotecluster/confirm_invite` rest api because they are not set up to send messages yet.  I'd rather not expose the ping APIs to the rest api layer.